taxi/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from .models import Car, Manufacturer
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = "__all__"

```

Comment:
Using `fields = "__all__"` will include all fields from the Car model in the form. Ensure that this is intentional and that no sensitive fields are exposed. Consider specifying only the fields you want to include explicitly.